### PR TITLE
:+1: Register a japan holiday in 2021

### PR DESCRIPTION
**Pull request by ryosukeYamazaki**
Japan 2021 holidays registered.

We are also making changes to the definition file.
See the PR https://github.com/holidays/definitions/pull/180.

I checked the output with the `make console` and compared it with the data published by the Cabinet Office on Japanese holidays.
https://www8.cao.go.jp/chosei/shukujitsu/gaiyou.html

Here are the commands we used in the test.

- [x] `make console`
- [x] `pp Holidays.year_holidays([:jp], Date.civil(2021, 1, 1)) `
- [x] compared with https://www8.cao.go.jp/chosei/shukujitsu/gaiyou.html

### CODE REVIEW

File: test/defs/test_defs_jp.rb [hunk 1]
```diff
@@ -115,7 +115,7 @@ def test_jp
 
     assert_equal "海の日", (Holidays.on(Date.civil(2020, 7, 23), [:jp])[0] || {})[:name]
 
-    assert_equal "海の日", (Holidays.on(Date.civil(2021, 7, 19), [:jp])[0] || {})[:name]
+    assert_equal "海の日", (Holidays.on(Date.civil(2021, 7, 22), [:jp])[0] || {})[:name]
```
Comment (ttwo32) [inline test/defs/test_defs_jp.rb:118]: Thank you for adding tests !! 👍

**Review by ttwo32 — APPROVED** [review]
Looks good.

**Comment by ttwo32** [comment]
@ryosukeYamazaki Thanks for your contributing !! 👍

**Comment by ryosukeYamazaki** [comment]
@ttwo32 Thanks for reviewing PR.

**Pull request merged**